merlin/models/tf/models/base.py

```diff
@@ -719,15 +787,6 @@ def evaluate(
             **kwargs,
         )
 
-    def compute_loss_metrics(
```

Comment:
looks like this is an unused function? and unrelated to the issue of throughput?

Reply:
Exactly @oliverholworthy . This is dead code that was used before Marc's refactory.